.travis.yml

```diff
@@ -10,6 +10,12 @@ install:
   - composer install
 
 script:
+  # check with lock file  
+  - ./vendor/bin/phpunit
+
+  # check without lock file  
+  - rm composer.lock && rm -rf vendor
+  - composer install
```

Comment:
This doesn't really make sense. It just tests whether the lock file can be re-created. Installing as a library is different as it doesn't respect pinned dependencies for example.